bpf/lib/lb.h

```diff
@@ -185,7 +185,11 @@ struct bpf_elf_map __section_maps LB_AFFINITY_MATCH_MAP = {
 };
 #endif
 
-#define REV_NAT_F_TUPLE_SADDR 1
+#define REV_NAT_F_TUPLE_SADDR	1
+#ifndef DSR_XLATE_MODE
+# define DSR_XLATE_MODE		0
```

Comment:
Nit: is spacing before `0` is off or GH is not able to render it properly?

Reply:
It's the rendering: https://github.com/cilium/cilium/blob/pr/xlate-l4/bpf/lib/lb.h#L187